addons/portal/static/src/scss/portal.scss

```diff
@@ -457,6 +457,7 @@ form label {
         width: 45px;
         height: 45px;
         margin-right: 1rem;
+        object-fit: cover;
```

Comment:
`cover` doesn't work well for this case because the image is resized to 50px before applying `cover`. cover should only be used when images aren't resized so much in the first place so that there are some pixels left after cropping.

I would suggest to remove the 50x50 from the template (or use `contain` instead, but as we typically use `cover` for avatars probably change the size).

Reply:
> `cover` doesn't work well for this case because the image is resized to 50px before applying `cover`. `cover` should only be used when images aren't resized so much in the first place so that there are some pixels left after cropping.
> 
> I would suggest to also remove the 50x50 from the template (or use `contain` instead, but as we typically use `cover` for avatars probably keep it and change the size too).

@seb-odoo done bro